### DISCUSSION TITLE
Preventing layers from versioning in Cloud

### DISCUSSION THREAD

**Discussion by eoyston** [opengisch/QField#4460] [Q&A]
Hi all,

Is there a way to prevent the versioning of your basemap in the cloud?
I have a 60mb raster that is the basemap for my project which is fine to stay static, but my Qfield Cloud repository seems to be versioning which then causes my 100mb quota to be exceeded and prevents me from syncing/using the project until i manually go remove a version through the account online.

Any suggestions?

Thanks

**Comment by xavierpuig** [comment]
Hi,
I encounter the same issue. In the QField menu from layer options I have tried the following, but new versions keep being created whenever I upload data from the tablet: 
- Cloud layer action: Directly access data source
- Cable layer action: Keep existent (Copy if missing) <- I thought this parameter would prevent the versioning, but apparently it doesn't.

Any guidance will be greatly appreciated. 

Best,

**Comment by jpasner** [comment]
Also having this issue. Would love the ability to toggle on and off versioning for any layer.  Love this App!
